src/common/options/rgw.yaml.in

```diff
@@ -3568,3 +3568,17 @@ options:
   - rgw
   - osd
   with_legacy: true
+- name: rgw_allow_notification_secrets_in_cleartext
+  type: bool
+  level: advanced
+  desc: Allows sending secrets (e.g. passwords) over non encrypted HTTP messages.
+  long_desc: When bucket notification ednpoint require secrets (e.g. passwords),
+    we allow the topic creation only over HTTPS messages. 
+    This parameter can be set to "true" to bypass this check.
+    Use this only if secure connection cannot be established with the radosgw
+    (e.g. a proxy in front of radosgw is used for ssl termination), and if the 
+    network through which these requests are sent is trusted. Otherwise, this will 
```

Comment:
`rgw_trust_forwarded_https` already covers the case where a proxy is terminating ssl. how about this wording?

> Use this only if radosgw is on a trusted private network, and the message broker cannot be configured without password authentication.